app/pages/profile/page.jsx

```diff
@@ -113,12 +116,19 @@ const ProfilePage = () => {
                 <h2 className="text-xl mt-2 md:mt-10 text-gray-900 w-full">Achievements</h2>
               </div>
               <div className='w-full flex flex-col'>
-              <div className="mt-4 w-full"> {/* This div will occupy the remaining space */}
-                  <Achievements title={'Commited Learner'} desc={'Reach a 3 day streak' } imm={CheckIcon2} total={3} progress={2} bcolor={"bg-orange-400"} />
-                </div>
-                <div className="mt-4 w-full"> {/* This div will occupy the remaining space */}
-                <Achievements title ={'Point Collector'}desc={'Earn 1800 more point' }  imm={CheckIcon3} total={3000} progress={1200} bcolor={"bg-lime-300"} />
-              </div>
+                {achiev
```

Comment:
since we don't have a user that has done any achievement yet we're getting all the achievements ( as you saw in the function we got the whole collection and get the data out of it ) , and just showed 2 of the documents in our page, this needs to be changed later after I add the functions to add the baught courses and watched links and login dates to our database //most likely we don't have time for that